HttpClient/SecondApp/src/app/post.ts

```diff
@@ -0,0 +1,6 @@
+export class Post {
+  id: number;
```

Comment:
```suggestion
  id?: number;
```
You can mark id as optional and use one model to get and to create posts